calico_upgrade/Makefile

```diff
@@ -0,0 +1,339 @@
+.PHONY: all binary test clean help
+default: help
+all: test                                            ## Run all the tests
+test: test-containerized                             ## Run all the tests
+all: dist/calico-upgrade dist/calico-upgrade-darwin-amd64 dist/calico-upgrade-windows-amd64.exe test-containerized
+
+###############################################################################
+# vX.Y format: update during major release process
+RELEASE_STREAM ?= master
+
+###############################################################################
+# Go Build versions
+GO_BUILD_VER:=v0.7
```

Comment:
suggest `v0.8` - most other components were bumped to @fasaxc 's v0.8 build container which has CVE fixes as well as golang 1.9 support.